shared/video-sdk/reference/_pricing.mdx

```diff
@@ -148,6 +148,30 @@ Agora offers a 10% overage allowance for all tiers of PCU at no additional charg
 
 Billing occurs at the end of each calendar month. For other tiers, contact [Agora Customer Support](mailto:support@agora.io).
 
+### AI Noise Suppression pricing
+
+[AI Noise Suppression](../enable-features/ai-noise-suppression) helps reduce noise and voice distortion in virtual communication.
+Agora charges for voice sent to the channel by the host while noise suppression is activated.
```

Comment:
while AI Noise Suppression is activated.

Reply:
It will be naming the product 3 times in 3 sentences in a row, but as you wish